frontend/src/custom-theme.scss

```diff
@@ -46,3 +46,19 @@ $theme: mat.define-light-theme(
 );
 
 @include mat.all-component-themes($theme);
+
+/* width */
+.simple-scroll::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Handle */
+.simple-scroll::-webkit-scrollbar-thumb {
+  background: grey;
+  border-radius: 10px;
+}
+
+.simple-scroll::-webkit-scrollbar-thumb:hover {
+  background: lightgrey;
+  border-radius: 10px;
+}
```

Comment:
Can we find an alternative to this approach? 

The docs (https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar) state:
"Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future."

Reply:
As far as I know, it's the only way to style a scrollbar. Simplebar also used this solution. I would advise to keep the default scrollbar for now

Reply:
I checked firefox, looks good too. The default style is fine.